### ISSUE TITLE
Invalid values for `aria-owns` and `aria-activedescendant`

### ISSUE THREAD

**Issue by birksy89** [ericgio/react-bootstrap-typeahead#438]
###  Intro
Hello, I did have a read through the other issues with similar titles - But none seem to exactly cover this issue.

I've had a look at the Class interface, and can see there is an "ID" type - But it's not showing up for me, and causing my markup to fail W3c validation.

PS - Thank you for this great module!

### Version
4.0.0-alpha.9

### Steps to reproduce

Set the "id" prop on "Typahead" component:

```
<Typeahead
          multiple
          id="CountrySearch"
          onChange={selected => {
            // Handle selections...
            onCountriesChange(selected);
          }}
          options={defaultCountries}
          placeholder="Choose a Location..."
        />
```

### Expected Behavior

"id" prop rendered in HTML
### Actual Behavior

"id" prop not showing in HTML:
![Capture](https://user-images.githubusercontent.com/4971630/56498237-bbfb8200-64f8-11e9-9c09-20a65818ef71.PNG)

### W3C Validation:
![w3c](https://user-images.githubusercontent.com/4971630/56498365-3c21e780-64f9-11e9-9746-a6c503479663.PNG)

**Comment by birksy89** [comment]
Hi @ericgio 

Yes, forgive me - I seem to have got myself in a knot...

The W3C validation is asking for a value to be present for:

- aria-activedescendant
- aria-owns

![image](https://user-images.githubusercontent.com/4971630/56679929-109c2a00-66be-11e9-9455-05901050bc6b.png)

It works... When the input has focus, and adds the following:
aria-owns="CountrySearch"

I've got myself confused, and have wrongly thought the input having an ID is the solution.

Going back to the original point - Is it possible to have the component pass the W3C validation?

**Comment by ericgio** [comment]
I see, thanks for the clarification. Yeah it looks like `aria-activedescendant` and `aria-owns` need to be set to `null` or `undefined` rather than an empty string when the component is unfocused or no menu items are active.

**Comment by ericgio** [comment]
Fixed in v3.4.3 and v4.0.0-alpha.10
